spec/workers/moderator/banish_user_worker_spec.rb

```diff
@@ -4,7 +4,7 @@
   include_examples "#enqueues_on_correct_queue", "high_priority", 1
 
   describe "#perform" do
-    let(:user) { create(:user, currently_hacking_on: "text is here") }
+    let(:user) { create(:user) }
```

Comment:
I completely removed the `currently_hacking_on` attribute here as it has nothing to do with the spec and we don't need it for any assertions.